packages/cli/src/__tests__/commands/locations.test.ts

```diff
@@ -0,0 +1,36 @@
+import { v4 as uuidv4 } from 'uuid'
+import { outputListing } from '@smartthings/cli-lib'
+import LocationsCommand from '../../commands/locations'
+
+
+jest.mock('@smartthings/cli-lib', () => {
+	const originalLib = jest.requireActual('@smartthings/cli-lib')
+
+	return {
+		...originalLib,
+		outputListing: jest.fn(),
+	}
+})
+
+describe('LocationsCommand', () => {
+	const mockListing = outputListing as unknown as jest.Mock<typeof outputListing>
```

Comment:
This seems required with Typescript in order to use anything useful in the jest mock API.

Reply:
You could probably do something like this:

```
const outputListingMock = jest.fn()
jest.mock('@smarthings/cli-lib', () => {
...
  return {
    ...originalLib,
    outputListing: outputListingMock,
  }
})
```
And then use `outputListingMock` instead.

Reply:
I tried that first and ran into [this issue](https://github.com/facebook/jest/issues/10996). It may be a temporary bug.